docs/configuration.md

```diff
@@ -22,4 +22,48 @@ For the complete list of settings GCM Core understands, see the list below.
 
 ## Available settings
 
-_Currently there are no available settings. This is a placeholder for the settings on the backlog._
+### credential.provider
+
+Define the host provider to use when authenticating.
+
+ID|Provider
+-|-
+`auto` _(default)_|_\[automatic\]_
+`azure-repos`|Azure Repos
+`github`|GitHub
+`generic`|Generic
```

Comment:
What would the basic equivalent be here?

Reply:
`generic`.. ish.

GCM Core doesn't support the ability to say "Don't use WIA/NTLM, even if the server supports it"

That should probably be added as an option?

Reply:
Added!